### PR TITLE
Dom updates

### PR DESCRIPTION
**Pull request by laurenlucero**
# Pull Request

### If it is a new feature, describe the feature and the iteration of the project spec it addresses. Note any questions or struggles as well.
Completed Iteration 1 and 2 of the spec. Improved login functionality, instantiated Hotel, Managers and Guest on login. Guest bookings and spending are displaying. Manager available rooms, revenue and occupancy are displaying. 
I am unsure of how to use the domUpdates object effectively. I have dom manipulation happening in my index.js because it relies on the instatiated classes to display information. My login/logout functionality was breaking when I tried to put it into functions so right now it is still out in the global scope. Feedback on how to refactor would be appreciated.

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:
- [X] My code follows the Turing style guidelines
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### Files Modified:
	modified:   src/Guest.js
	modified:   src/css/base.scss
	modified:   src/index.html
	modified:   src/index.js

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -1,39 +1,50 @@
...
+    $("#login-holder").css("display", "none");
+    $(".manager-dashboard").removeClass("hide");
+  },
+
+  displayGuestDash() {
+    $("#login-holder").css("display", "none");
+    $(".guest-dashboard").removeClass("hide");
+  },
+
+  logout() {
```
Comment (Karlfunhouse) [inline src/domUpdates.js:18]: woah i didn't know you could modify css like this!  Awesome

File: src/index.html [hunk 2]
```diff
@@ -15,85 +15,105 @@
...
+          action="https://fe-apps.herokuapp.com/api/v1/overlook/1904/bookings/bookings"
+          method="post"
+        >
+          <label for="select-date">Select date to view available rooms:</label>
+          <input
+            type="date"
+            id="select-date"
+            name="trip-date"
+            value="yyyy-MM-dd"
+            min="2020-01-01"
```
Comment (Karlfunhouse) [inline src/index.html:89]: nice min and max I should do that too.

File: src/index.js [hunk 3]
```diff
@@ -1,82 +1,156 @@
...
+      return 0;
+    })
+    .map(booking => {
+      $(".my-bookings").append(
+        `<li>Date: ${booking.date}, Room: ${booking.roomNumber}</li>`
+      );
+    });
+};
+
+const displayGuestSpending = () => {
```
Comment (Karlfunhouse) [inline src/index.js:123]: do you plan on moving all these display methods into your domUpdates class?

**Comment by laurenlucero** [comment]
@Karlfunhouse @hannahhch @thatpamiam

**Pull request merged**